tokens.xml

```diff
@@ -7058,7 +7058,7 @@ Equip {1}</text>
                 <cmc>0</cmc>
                 <pt>1/1</pt>
             </prop>
-            <set picURL="https://media-dominaria.cursecdn.com/attachments/113/426/635201279747658553.jpg">C13</set>
+            <set picURL="https://c1.scryfall.com/file/scryfall-cards/large/front/1/3/13e4832d-8530-4b85-b738-51d0c18f28ec.jpg?1630007268">C13</set>
```

Comment:
this is a low quality scan, I think it's just the wizards token but upscaled

Reply:
That's all they have currently:
https://api.scryfall.com/cards/13e4832d-8530-4b85-b738-51d0c18f28ec

While being a bit bigger in resolution and way bigger in size... the png seems to be the same?

Reply:
actually, after I've merged this, was this token actually printed in C13?

Reply:
It's printed in CC2 [as far as I'm aware](https://github.com/Cockatrice/Magic-Token/issues/140#issuecomment-1106369825).